README.md

```diff
@@ -444,8 +445,9 @@ When you use OAuth 2.0, the full YAML configuration looks like:
 ```yaml
 okta:
   client:
-    connectionTimeout: 30 # seconds
-    orgUrl: "https://{yourOktaDomain}" # i.e. https://dev-123456.oktapreview.com
+    connectionTimeout: 30               # seconds
+    jwtExpiryTime: 3600                 # seconds
```

Comment:
This key name should be defined in the Okta SDK technical spec first, as so other Okta SDKs use the same name.
cc: @arvindkrishnakumar-okta

Reply:
IMHO, it should NOT contain "jwt" as that is an implementation detail, but it may need to be moved under a new node, for example `okta.client.oauth.*` which would may contain a `kid` and `tokenTtl`

NOTE: in other places, we use `Timeout`, so we need to be careful here to make sure we either stay consistent, or use a term that doesn't overlap.

Reply:
This might also be a great time to think about using a ISO 8601 based duration (defaulting to the unit of seconds) if the value is a simple number. (cc: @bretterer)

Reply:
@VitaliiTytarenko-okta If we're introducing this new config parameter, we need to propose a spec change at https://github.com/okta/oss-technical-designs/blob/master/technical_designs/oauth-in-management-sdks.md in order for all backend SDKs to be consistent. Can you create a new PR in the above spec and detail the change?